web/js/tasks.js

```diff
@@ -890,7 +906,7 @@ Ext.onReady(function(){
             },
         });
 
-        if(freshCreatedTask) {
+        if(typeof(freshCreatedTask) === 'boolean') {
```

Comment:
You are only checking if `freshCreatedTask` is a boolean, but not if it's true! You should do both checks, otherwise it won't work in case you call newTask(false) :)